core/src/test/scala/other/kafka/ReplicationQuotasTestRig.scala

```diff
@@ -154,6 +154,19 @@ object ReplicationQuotasTestRig {
       println("Output can be found here: " + journal.path())
     }
 
+    def getReassignmentJson(partitionsToBeReassigned: Map[TopicPartition, Seq[Int]]): String = {
```

Comment:
This is a util method to convert Map to json string. This is only used in this test class. To avoid dependency on ZKutils class, I copied the method here. I can revert this change, later we can do this cleanup, when we remove Zkutils.

Reply:
Yeah, I'd prefer to use `ZkUtils` than duplicate serialization logic.